rosidl_typesupport_introspection_cpp/include/rosidl_typesupport_introspection_cpp/MessageIntrospection.h

```diff
@@ -6,7 +6,7 @@
 
 #include <rosidl_generator_c/message_type_support.h>
 
-#include "impl/visibility_control.h"
+#include "visibility_control.h"
```

Comment:
I missed that it was already moved out of the impl folder. I thought it is found because the impl folder is exported.

In general should be use "absolute" imports in these cases rather then relative imports instead (in this case `#include "rosidl_typesupport_introspection_cpp/visibility_control.h"`)? Kind of what is also recommended in Python.